update-script.js

```diff
@@ -0,0 +1,141 @@
+const KintoClient = require("kinto-http").default;
+const btoa = require("btoa");
+const fetch = require("node-fetch");
+const AppConstants = require("./app-constants");
+
+const WEBSITES_WITH_SHARED_CREDENTIAL_COLLECTION = "websites-with-shared-credential-backends";
+/** @type {String} */
+const FX_RS_WRITER_USER = AppConstants.FX_REMOTE_SETTINGS_WRITER_USER;
+/** @type {String} */
+const FX_RS_WRITER_PASS = AppConstants.FX_REMOTE_SETTINGS_WRITER_PASS;
+/** @type {String} */
+const SERVER_ADDRESS = AppConstants.FX_REMOTE_SETTINGS_WRITER_SERVER;
+
+const APPLE_API_ENDPOINT = "https://api.github.com/repos/apple/password-manager-resources/contents/quirks/websites-with-shared-credential-backends.json";
+
+/**
+ * Fetches the source records from the APPLE_API_ENDPOINT
+ *
+ * @return {String[][]} 
+ */
+const getSourceRecords = async () => {
+  const response = await fetch(APPLE_API_ENDPOINT, {
+    headers: {
+      "Accept": "application/vnd.github.v3.raw"
+    }
+  });
+  const data = await response.json();
+  return data;
+}
+
+const arrayEquals = (a, b) => {
+  return Array.isArray(a) &&
+    Array.isArray(b) &&
+    a.length === b.length &&
+    a.every((val, index) => val === b[index]);
+};
+
+/**
+ * Updates the existing record in Remote Settings with the updated data from Apple's GitHub repository
+ *
+ * @param {KintoClient} client KintoClient instance
+ * @param {string} bucket Name of the Remote Settings bucket
+ * @param {Object} newRecord Object containing the updated related realms object
+ * @param {string} newRecord.id ID from the current related realms object from the Remote Settings server
+ * @param {string[][]} newRecord.relatedRealms Updated related realms array from GitHub
+ */
+const updateRecord = async (client, bucket, newRecord) => {
+  await client.bucket(bucket).collection(WEBSITES_WITH_SHARED_CREDENTIAL_COLLECTION).updateRecord(newRecord);
```

Comment:
I can't remember in which cases the `updateRecord()` would throw an error. Maybe a comment would help to recall why we don't read the returned value.

Reply:
I don't quite remember when an error would be thrown...I think it's primarily if the collection doesn't exist or if the user doesn't have permissions to the collection. I'll do some wrangling to figure that out.

Reply:
Since you have a global `try/catch` you should be fine

Reply:
sounds good to me 👍